bower.json

```diff
@@ -7,6 +7,7 @@
     "jquery-details": "https://github.com/mathiasbynens/jquery-details/archive/v0.1.0.tar.gz",
     "digitalmarketplace-frontend-toolkit": "https://github.com/alphagov/digitalmarketplace-frontend-toolkit.git#v23.0.1",
     "govuk_template": "https://github.com/alphagov/govuk_template/releases/download/v0.19.2/jinja_govuk_template-0.19.2.tgz",
-    "digitalmarketplace-frameworks": "https://github.com/alphagov/digitalmarketplace-frameworks.git#8.10.0"
+    "digitalmarketplace-frameworks": "https://github.com/alphagov/digitalmarketplace-frameworks.git#8.10.0",
+    "scrolldepth": "https://github.com/robflaherty/jquery-scrolldepth.git#1.0"
```

Comment:
Although we haven't done this yet, we should probably fork this into alphagov and pull it from there, as a third party could pull this repo at any time and break our deploys.